pkgs/racket-doc/scribblings/guide/contracts/new-combinators.scrbl

```diff
@@ -179,23 +182,21 @@ to use this API looks like this:
        v
        '(expected "an integer" given: "~e")
        v)))
-  (λ (f)
+  (λ (f neg-party)
     (if (and (procedure? f)
              (procedure-arity-includes? f 1))
-        (λ (neg-party)
-          (λ (x)
-            (check-int x dom-blame neg-party)
-            (define ans (f x))
-            (check-int ans rng-blame neg-party)
-            ans))
-        (λ (neg-party)
-          (raise-blame-error
-           blame #:missing-party neg-party
-           f
-           '(expected "a procedure of one argument" given: "~e")
-           f)))))]
+        (λ (x)
+          (check-int x dom-blame neg-party)
+          (define ans (f x))
+          (check-int ans rng-blame neg-party)
+          ans)
+        (raise-blame-error
+         blame #:missing-party neg-party
+         f
+         '(expected "a procedure of one argument" given: "~e")
+         f))))]
 The advantage of this style of contract is that the @racket[_blame]
-and @racket[_v] arguments can be supplied on the server side of the
+and @racket[_f] arguments can be supplied on the server side of the
```

Comment:
I think this part might need to change: it looks like the `f` argument has to be supplied for each client, but I'm not very confident.